demos/serverless/README.md

```diff
@@ -2,7 +2,7 @@
 
 This demo shows how to deploy [Chime SDK Browser Demo](https://github.com/aws/amazon-chime-sdk-js/tree/master/demos/browser) as self-contained serverless applications.
 
-> *Note: deploying the serverless demo may incur extra charges in your AWS account.*
+> *Note: Deploying the Amazon Chime SDK demo applications contained in this repository will cause your AWS Account to be billed for services, including the Amazon Chime SDK, used by the application.*
```

Comment:
Should we lowercase 'Account' and add "AWS" before services in the second line?

Reply:
I just copied and pasted from the wording in the ticket. Approved by legal